### PR TITLE
[2.6.x]: Make it clear where to add the sbt-eclipse plugin dependency

### PR DESCRIPTION
**Pull request by ManognaMutyala**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [ ] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [ ] Have you added copyright headers to new files?
* [ ] Have you checked that both Scala and Java APIs are updated?
* [ ] Have you updated the documentation for both Scala and Java sections?
* [ ] Have you added tests for any changed functionality?

## Purpose
To make documentation more comprehensible

What does this PR do?
It provides the  newbie and starters about actual file in which plugin has to be added.

### CODE REVIEW

File: documentation/manual/gettingStarted/IDE.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ However, using a modern Java or Scala IDE provides cool productivity features li
 
 ### Setup sbteclipse
 
-Integration with Eclipse requires [sbteclipse](https://github.com/typesafehub/sbteclipse). Make sure to always use the [most recent available version](https://github.com/typesafehub/sbteclipse/releases).
+Integration with Eclipse requires [sbteclipse](https://github.com/typesafehub/sbteclipse). Make sure to always use the [most recent available version](https://github.com/typesafehub/sbteclipse/releases).Add the below code to the plugin.sbt file.
```
Comment (gmethvin) [inline documentation/manual/gettingStarted/IDE.md]: Should be something like: "Make sure to always use the most recent version in your `project/plugins.sbt` file:"

**Comment by gmethvin** [comment]
We also would need a similar change to master. Normally we ask to submit changes to master unless it's only relevant on specific version(s).

File: documentation/manual/gettingStarted/IDE.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ However, using a modern Java or Scala IDE provides cool productivity features li
 
 ### Setup sbteclipse
 
-Integration with Eclipse requires [sbteclipse](https://github.com/typesafehub/sbteclipse). Make sure to always use the [most recent available version](https://github.com/typesafehub/sbteclipse/releases).
```
Comment (marcospereira) [inline documentation/manual/gettingStarted/IDE.md]: The markdown syntax is broken here. This is valid:

```markdown
[text to show](http://my-link.com)
```
This won't work:
```markdown
[text to show] some more text (http://my-link.com)
```
In other words, keep the link text and the link URL together. Your change needs to be:

```markdown
... [most recent available version](https://github.com/typesafehub/sbteclipse/releases) in your project/plugins.sbt file.
```

Comment (ManognaMutyala) [reply documentation/manual/gettingStarted/IDE.md]: Thanks Sir!! I have corrected the PR as you suggested.

File: documentation/manual/gettingStarted/IDE.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ However, using a modern Java or Scala IDE provides cool productivity features li
...
+Integration with Eclipse requires [sbteclipse](https://github.com/typesafehub/sbteclipse). Make sure to always use the              [most recent available version](https://github.com/typesafehub/sbteclipse/releases) in your project/plugins.sbt file.
```
Comment (benmccann) [inline documentation/manual/gettingStarted/IDE.md]: It doesn't necessarily need to be in project/plugins.sbt. E.g. you can also put it in ~/.sbt/0.13/plugins/

Comment (marcospereira) [reply documentation/manual/gettingStarted/IDE.md]: Watch for the extra spaces here, after "the":

```markdown
the              [most recent available version](https://github.com/typesafehub/sbteclipse/releases) in
```

**Comment by marcospereira** [comment]
Thank you, @ManognaMutyala.

File: documentation/manual/gettingStarted/IDE.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ However, using a modern Java or Scala IDE provides cool productivity features li
 
 ### Setup sbteclipse
 
-Integration with Eclipse requires [sbteclipse](https://github.com/typesafehub/sbteclipse). Make sure to always use the [most recent available version](https://github.com/typesafehub/sbteclipse/releases).
+Integration with Eclipse requires [sbteclipse](https://github.com/typesafehub/sbteclipse). Make sure to always use the [most recent available version](https://github.com/typesafehub/sbteclipse/releases) in your project/plugins.sbt file or follow [sbteclipse docs](https://github.com/typesafehub/sbteclipse#for-sbt-013-and-up) to install globally.
 
 ```scala
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.2.2")
```
Comment (PromanSEW) [inline documentation/manual/gettingStarted/IDE.md]: Plugin has version 5.2.3 now

**Pull request merged**